numpy/random/mtrand/mtrand.pyx

```diff
@@ -1468,13 +1468,14 @@ cdef class RandomState:
             less than high.  The default value is 1.0.
         size : int or tuple of ints, optional
             Output shape.  If the given shape is, e.g., ``(m, n, k)``, then
-            ``m * n * k`` samples are drawn.  Default is None, in which case a
-            single value is returned.
+            ``m * n * k`` samples are drawn.  If size is ``None`` (default),
```

Comment:
I guess we should modify the description of `low` to `float or array-like of floats, optional`, shouldn't we?

The same of course applies to all other broadcastable arguments.